### DISCUSSION TITLE
Is it possible to adding api key for each clients ?

### DISCUSSION THREAD

**Discussion by eguvenc** [verdaccio/verdaccio#3884] [Q&A]
Hi everyone, 
I want to set connection limits to my clients by creating an api key for each of them.
Is this possible ?
If yes, could someone show me an example how can i do that.

For instance 
```
npm adduser --registry https://example.dev:4873/my-clients-api-key
```
Thanks.

**Comment by juanpicado** [comment]
From my side, unfortunately I haven't get any info how those are using verdaccio in that way how they achieve it, usually is private source code, anyhow there is no internal default feature to achieve that.
